test/integration/golden_response/get_stat_ranking/country.json

```diff
@@ -20,1216 +20,1246 @@
           },
           {
-            "rank": 40,
+            "rank": 41,
             "value": 37589262,
             "placeDcid": "country/CAN"
           },
           {
-            "rank": 41,
+            "rank": 42,
             "value": 36471769,
             "placeDcid": "country/MAR"
           },
           {
-            "rank": 42,
+            "rank": 43,
             "value": 34268528,
             "placeDcid": "country/SAU"
           },
           {
-            "rank": 43,
+            "rank": 44,
             "value": 33580650,
             "placeDcid": "country/UZB"
           },
           {
-            "rank": 44,
+            "rank": 45,
             "value": 31949777,
             "placeDcid": "country/MYS"
           },
           {
-            "rank": 45,
+            "rank": 46,
             "value": 31826000,
             "placeDcid": "country/PER"
           },
           {
-            "rank": 46,
+            "rank": 47,
             "value": 31825295,
             "placeDcid": "country/AGO"
           },
           {
-            "rank": 47,
+            "rank": 48,
             "value": 30417856,
             "placeDcid": "country/GHA"
           },
           {
-            "rank": 48,
+            "rank": 49,
             "value": 30366036,
             "placeDcid": "country/MOZ"
           },
           {
-            "rank": 49,
+            "rank": 50,
             "value": 29161922,
             "placeDcid": "country/YEM"
           },
           {
-            "rank": 50,
+            "rank": 51,
             "value": 28608710,
             "placeDcid": "country/NPL"
           },
           {
-            "rank": 51,
+            "rank": 52,
             "value": 28515829,
             "placeDcid": "country/VEN"
           },
           {
-            "rank": 52,
+            "rank": 53,
             "value": 26969307,
             "placeDcid": "country/MDG"
           },
           {
-            "rank": 53,
+            "rank": 54,
             "value": 25876380,
             "placeDcid": "country/CMR"
           },
           {
-            "rank": 54,
+            "rank": 55,
             "value": 25716544,
             "placeDcid": "country/CIV"
           },
           {
-            "rank": 55,
+            "rank": 56,
             "value": 25666161,
             "placeDcid": "country/PRK"
           },
           {
-            "rank": 56,
+            "rank": 57,
             "value": 25364300,
             "placeDcid": "country/AUS"
           },
           {
-            "rank": 57,
+            "rank": 58,
             "value": 23568378,
             "placeDcid": "country/TWN"
           },
           {
-            "rank": 58,
+            "rank": 59,
             "value": 23310715,
             "placeDcid": "country/NER"
           },
           {
-            "rank": 59,
+            "rank": 60,
+            "value": 23229846,
+            "placeDcid": "country/YUG"
```

Comment:
ditto for yugoslavia (and other deprecated countries)

Reply:
cl/379660649 should fix the bug.